AAUnlimited/MemMods/Shared/Events/GameTick.cpp

```diff
@@ -71,6 +72,28 @@ void CheckRoomChange() {
 	}
 }
 
+void hPositionChange() {
+	auto hPositionValue = Shared::GameState::getHPosition();
+	const DWORD offset[]{ 0x3761CC, 0x28, 0x38, 0x5F4 };
+	DWORD* hPosition = (DWORD*)ExtVars::ApplyRule(offset);
+	if (hPosition != nullptr) {
```

Comment:
are you sure it's `nullptr` when you go to title screen? It can be not null but still invalid so the next line would crash